changelog/unreleased/issue-21720.toml

```diff
@@ -0,0 +1,5 @@
+type = "f"
+message = "Fixed broken filtering on Dashboards page."
```

Comment:
Can you make this a bit more precise and mention which parts of filtering were broken?